src/aspen/aspen.api/appsettings.json

```diff
@@ -9,5 +9,6 @@
       "Microsoft.Hosting.Lifetime": "Information"
     }
   },
-  "AllowedHosts": "*"
+  "AllowedHosts": "*",
+  "PGPASSFILE": "C:\\Users\\dj.bigelow\\Source\\repos\\project_aspen\\src\\aspen\\.pgpass"
```

Comment:
`"PGPASSFILE": "../.pgpass"`

Wouldn't that work?  You don't want to have a full path in there, it will break on anyone else's machine.

Reply:
I'm pretty sure it would and we should change it, but in our defense that was for us to run it manually so the path there isn't needed for running on docker( The pgpassfile is given as an environment variable in docker compose so to get it running manually I had to change the startup file to work with the appsettings instead of a given docker environment).